panel/src/main/scala/hmda/panel/PanelCsvParser.scala

```diff
@@ -43,25 +44,36 @@ object PanelCsvParser extends InstitutionComponent {
       sys.exit(1)
     }
 
-    val timeout = 5.seconds
-
     println("Cleaning DB...")
-    Await.result(repository.dropSchema(), timeout)
+    Await.result(repository.dropSchema(), 5.seconds)
```

Comment:
There's an `implicit val timeout` declared on line 31, it was causing some compilation issues.